docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst

```diff
@@ -473,38 +471,235 @@ and add the Run Script definition to the job intended to backup the data for a R
+important information you may need in an emergency:
+
+-  What is the most recent backup of the ``BackupCatalog`` job?
+
+-  What medium was this backup written to?
+
+Ways to approach a restore
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The restore procedure depends on the following factors:
+
+-  How badly damaged is the Bareos system?
+
+-  What data are available for the restore?
+
+In an emergency restore, the first objective is to restore the catalog
+database. As soon as the catalog database is restored, all other data
+and their storage locations are also available.
+
+Of course an emergency restore of the Bareos system is only possible if
+you have storage media containing the necessary data. The degree of
+destruction and available restore data determine the restore method. The
+aim is always to get the backup system working again in a way as
+efficient as possible. Less damage and more available restore data make this
+stage quicker to achieve.
+
+Tools
+~~~~~
+
+Next to programs for regular operation (director, file daemon, storage daemon,
+Bareos console) a Bareos installation contains additional service programs.
+Programs that work with media require a valid storage daemon configuration
+because they use the same routines to access storage media as the storage
+daemon does in normal operation. Other programs require the configuration file
+of the director, as they assess the configuration settings stored there.
+
+It is therefore a great advantage if the configuration files of your
+installation are available in an emergency. Otherwise, an emergency
+restore must be preceded by the restoring of the configuration.
+In most cases you will lack the time to do this properly.
+
+.. warning::
+
+   Conclusion keep a copy at least of the storage daemon able to read
+   the disaster recovery volume in an accessible format and location.
+
+
+``bextract`` allows you to extract data directly from Bareos media. It
+need a functioning storage daemon configuration file.
+
+``bls`` displays the content of Bareos media. It can access both tapes
+and files in the file system. It need a functioning storage daemon
+configuration file.
+
+
+Performing an emergency restore
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following section we will assume that our backup system itself
+has failed. We need the following to restore it:
+
+Preparation steps :
+
+-  The media containing the backup. If the backup itself has been
+   destroyed, it is of course no longer possible to restore it. You
+   should therefore always swap out backups in some form or at least
+   separate them from the actual backup system.
+
+-  The configuration of the storage daemon. If the properties of the
+   replacement system have changed (paths or SCSI devices, for example),
+   you have to adjust the storage daemon configuration to suit the new
+   environment.
+
+-  The ``Bootstrap`` file of the most recent backup of the database,
+   i.e. of the ``BackupCatalog`` job.
+
```

Comment:
Do we maybe also say that you need to install the bareos software including all tools and the required sd backends?

Reply:
You make a point here, we might want to have explicit steps, and as it is so obvious for us raise the flag it is not for newcomers